### DISCUSSION TITLE
Would an autoupdating "last_login" field on the User model be useful to anyone?

### DISCUSSION THREAD

**Discussion by taylor-shift** [fastapi-users/fastapi-users#796] [Ideas]
I have implemented this feature in my own project (using fastapi_users) and was wondering if anyone else would have a use for it.

If so I will submit PR.

Thanks.

**Comment by frankie567** [comment]
Hi @luisroel91!

What could be nice is that you share it in the "Show and tell" discussion category; so that other developers can get inspiration from what you did 🙂

Cheers!

**Reply by taylor-shift** [reply]
Will do Frankie.

Thanks for the great software.
